aws/src/test/java/org/apache/iceberg/aws/TestAwsProperties.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.apache.iceberg.aws;
-
-import java.io.IOException;
-import java.util.Collections;
-import org.apache.iceberg.TestHelpers;
-import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
-import org.assertj.core.api.Assertions;
-import org.junit.jupiter.api.Test;
-
-public class TestAwsProperties {
```

Comment:
why is this being removed?

Reply:
It was testing only deprecated `awsProperties.httpClientProperties()`, since it is removed. There is no need of the test class. 

Now we use `HttpClientProperties` instead which is already covered by `HttpClientPropertiesTest`

Reply:
it seems the purpose of this test was to check that Kryo ser/de works. Therefore we should keep this test (although using non-deprecated properties)

Reply:
ok. Added a test to verify some basic properties.